libcaf_core/caf/response_promise.hpp

```diff
@@ -105,6 +106,7 @@ class CAF_CORE_EXPORT response_promise {
   }
 
   /// Satisfies the promise by sending an error response message.
+  /// For non-requests, nothing is done.
```

Comment:
This isn't true. It would mean that CAF silently drops errors instead of propagating them for asynchronous messages.

```suggestion
```